uint8 id
geometry_msgs/PoseStamped pose_stamped